monix-execution/shared/src/main/scala/monix/execution/Scheduler.scala

```diff
@@ -409,7 +409,6 @@ object Scheduler extends SchedulerCompanionImpl {
   val TRACING = Features.flag(2)
 
   /** Utilities complementing the `Scheduler` interface. */
-  @deprecated("Extension methods are now implemented on `Scheduler` directly", "3.4.0")
```

Comment:
Is this accidental or on purpose?

Reply:
It's on purpose. The Scala 3 compiler ends up triggering that warning even when those extensions methods aren't used, don't know why.

Deprecating just the methods is enough IMO. Who knows, we might want to add other extensions in the future.